ccx_messaging/consumers/kafka_consumer.py

```diff
@@ -87,9 +85,7 @@ def __init__(
         self.dead_letter_queue_topic = dead_letter_queue_topic
 
         if self.dead_letter_queue_topic is not None:
-            config = translate_kafka_configuration(kafka_broker_config)
-            dlq_producer_config = producer_config(config)
-            self.dlq_producer = KafkaProducer(**dlq_producer_config)
+            self.dlq_producer = Producer(kwargs)
```

Comment:
for consistency we could call this ConfluentProducer like the consumer... but I'm probably being pedantic...

Reply:
I don't usually like to rename the imported classes, like I did with `Consumer` because that name collided with the Insights Core Messaging `Consumer`.

In any case, I would prefer to change the ICM import to "ICMConsumer" and restore KafkaConsumer to Consumer instead of changing Producer.

I'll take note to modify it in other tasks related to clean up the library code. Thank you!